internal/cmd/api-key/command_test.go

```diff
@@ -263,7 +263,7 @@ func (suite *APITestSuite) SetupTest() {
 					Id:          myServiceAccountId,
 					ResourceId:  myUserResourceId,
 					ServiceName: myAccountName,
-					Email:       "csreesangkom@confluent.io",
+					Email:       "cli-mock-email@confluent.io",
```

Comment:
How does changing the name here fix these tests? Do the tests still test what we intend them to?

Reply:
Since `getAllUsers` now includes the current user, the current user actually has the same resource id as this user, so it gets overridden. 

We could also fix this test by changing `myUserResourceId` to something else.

Reply:
Got it, cool!